packages/playwright/src/matchers/toMatchSnapshot.ts

```diff
@@ -206,7 +206,7 @@ class SnapshotHelper<T extends ImageComparatorOptions> {
       return this.createMatcherResult(message, true);
     }
     if (this.updateSnapshots === 'missing') {
-      this.testInfo._failWithError(serializeError(new Error(message)), false /* isHardError */);
+      this.testInfo._failWithError(new Error(message), false /* isHardError */);
```

Comment:
Is this the last caller outside of `TestInfo`? I wonder if we can do something here as well...

Reply:
I think it is the last one, but I am not sure there is something we can do.